src/main/haskell/kore/src/Kore/IndexedModule/MetadataTools.hs

```diff
@@ -24,7 +24,8 @@ import Kore.IndexedModule.Resolvers
 -- |'MetadataTools' defines a dictionary of functions which can be used to
 -- access the metadata needed during the unification process.
 data MetadataTools level attributes = MetadataTools
-    { attributes :: SymbolOrAlias level -> attributes
+    { symAttributes :: SymbolOrAlias level -> attributes
```

Comment:
`symAttributes` seems a little vague;  one could also read it as symbolic attributes.
Maybe rename it to `symbolAttributes` or `headAttributes` ?

Reply:
I think we can agree that sym = kore symbol in this codebase, idk of anything called "Symbolic" outside of SBV's API.